firestore-translate-text/functions/lib/logs/index.js

```diff
@@ -16,68 +16,68 @@
+    firebase_functions_1.logger.log(messages_1.messages.documentUpdatedUnchangedInput());
 };
 exports.error = (err) => {
-    console.error(...messages_1.messages.error(err));
+    firebase_functions_1.logger.error(...messages_1.messages.error(err));
 };
 exports.fieldNamesNotDifferent = () => {
-    console.error(messages_1.messages.fieldNamesNotDifferent());
+    firebase_functions_1.logger.error(messages_1.messages.fieldNamesNotDifferent());
 };
-exports.init = () => {
-    console.log(...messages_1.messages.init(config_1.default));
+exports.init = (config) => {
+    firebase_functions_1.logger.log(...messages_1.messages.init(config));
 };
 exports.inputFieldNameIsOutputPath = () => {
-    console.error(messages_1.messages.inputFieldNameIsOutputPath());
+    firebase_functions_1.logger.error(messages_1.messages.inputFieldNameIsOutputPath());
 };
 exports.start = () => {
-    console.log(...messages_1.messages.start(config_1.default));
+    firebase_functions_1.logger.log(messages_1.messages.start());
 };
 exports.translateInputString = (string, language) => {
-    console.log(messages_1.messages.translateInputString(string, language));
+    firebase_functions_1.logger.log(messages_1.messages.translateInputString(string, language));
 };
 exports.translateStringComplete = (string, language) => {
-    console.log(messages_1.messages.translateStringComplete(string, language));
+    firebase_functions_1.logger.log(messages_1.messages.translateStringComplete(string, language));
 };
 exports.translateStringError = (string, language, err) => {
-    console.error(...messages_1.messages.translateStringError(string, language, err));
+    firebase_functions_1.logger.error(...messages_1.messages.translateStringError(string, language, err));
 };
 exports.translateInputStringToAllLanguages = (string, languages) => {
-    console.log(messages_1.messages.translateInputStringToAllLanguages(string, languages));
+    firebase_functions_1.logger.log(messages_1.messages.translateInputStringToAllLanguages(string, languages));
 };
 exports.translateInputToAllLanguagesComplete = (string) => {
-    console.log(messages_1.messages.translateInputToAllLanguagesComplete(string));
+    firebase_functions_1.logger.log(messages_1.messages.translateInputToAllLanguagesComplete(string));
 };
 exports.translateInputToAllLanguagesError = (string, err) => {
-    console.error(...messages_1.messages.translateInputToAllLanguagesError(string, err));
+    firebase_functions_1.logger.error(...messages_1.messages.translateInputToAllLanguagesError(string, err));
 };
 exports.updateDocument = (path) => {
-    console.log(...messages_1.messages.updateDocument(path));
+    firebase_functions_1.logger.log(messages_1.messages.updateDocument(path));
 };
 exports.updateDocumentComplete = (path) => {
-    console.log(...messages_1.messages.updateDocumentComplete(path));
+    firebase_functions_1.logger.log(messages_1.messages.updateDocumentComplete(path));
```

Comment:
Is there a reason we are using a variable name with a suffix of _1.

Also can logger be deconstructed? I think this is available in Node 10?

```js 
const {logger} require("firebase-functions");
```